commands.md

```diff
@@ -232,28 +234,45 @@ This will dedent the current indentation to the root marked by `markAsRoot`.
 ### trim
 
 ```ts
-declare var trim: Doc;
+declare const trim: Doc;
 ```
 
 This will trim any whitespace or tab character on the current line. This is used for preprocessor directives.
 
 ### hardlineWithoutBreakParent and literallineWithoutBreakParent
 
-These are used very rarely, for advanced formatting tricks.
+_Added in v2.3.0_
+
+```ts
+declare const hardlineWithoutBreakParent: Doc;
+declare const literallineWithoutBreakParent: Doc;
+```
+
+These are used very rarely, for advanced formatting tricks. Unlike their "normal" counterparts, they don't include an implicit `breakParent`.
 
 Examples:
 
 - `hardlineWithoutBreakParent` is used for printing tables in Prettier's Markdown printer. With `proseWrap` set to `never`, the columns are aligned only if none of the rows exceeds `printWidth`.
-- `literallineWithoutBreakParent` (called `literalLineNoBreak` [there](https://github.com/prettier/plugin-ruby/blob/9e658db13d3f0524d816528e060b1381e0431559/src/ruby/nodes/heredocs.js)) is used in the [Ruby plugin](https://github.com/prettier/plugin-ruby) for printing heredoc syntax.
+- `literallineWithoutBreakParent` is used in the [Ruby plugin](https://github.com/prettier/plugin-ruby) for [printing heredoc syntax](https://github.com/prettier/plugin-ruby/blob/master/src/ruby/nodes/heredocs.js).
```

Comment:
```suggestion
- `literallineWithoutBreakParent` is used in the [Ruby plugin](https://github.com/prettier/plugin-ruby) for [printing heredoc syntax](https://github.com/prettier/plugin-ruby/blob/b6e7bd6bc3f70de8f146aa58ad0c8310518bf467/src/ruby/nodes/heredocs.js).
```

Better use a permalink.

Reply:
Or maybe link to line 2?

Reply:
The file is small enough and contains good comments